### PR TITLE
DOC: elaborate on copies vs in place operations in comparison docs

### PR DESCRIPTION
**Pull request by afeld**
Screen Shot 2021-01-06 at 12 15 05 AM

- [ ] ~~closes #xxxx~~
- [ ] tests added / passed
- [ ] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] ~~whatsnew entry~~

### CODE REVIEW

File: doc/source/getting_started/comparison/includes/copies.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+      df = df.sort_values("col1")
+
+* Assign to a new variable
+
+   .. code-block:: python
+
+      sorted_df = df.sort_values("col1")
+
+* Use the ``inplace=True`` keyword argument, where available
```
Comment (jreback) [inline doc/source/getting_started/comparison/includes/copies.rst:16]: frown on this 

in place is never recommended

**Review by jreback — CHANGES_REQUESTED** [review]

File: doc/source/getting_started/comparison/includes/copies.rst (hunk 1 above)
Comment (afeld) [reply doc/source/getting_started/comparison/includes/copies.rst:16]: A few options:

- Say "in order of preference"
- Mention `inplace` as "you will come across this, but don't use it"
- Remove mention entirely

Strong feelings?

Comment (jreback) [reply doc/source/getting_started/comparison/includes/copies.rst:16]: i ok with mentioning it, but put it in a note and point to this link: https://pandas.pydata.org/pandas-docs/dev/user_guide/indexing.html#indexing-view-versus-copy

**Comment by jreback** [comment]
also pls merge master

File: doc/source/getting_started/comparison/includes/copies.rst (hunk 1 above)
Comment (afeld) [reply doc/source/getting_started/comparison/includes/copies.rst]: Done, though the "Returning a view versus a copy" section doesn't actually mention `inplace`. I'd submit a pull request to add an explanation, but frankly, that's one area of pandas that's still pretty fuzzy to me.

Comment (jreback) [reply doc/source/getting_started/comparison/includes/copies.rst]: sure that would be great

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks !

File: doc/source/getting_started/comparison/includes/copies.rst (hunk 1 above)
Comment (afeld) [reply doc/source/getting_started/comparison/includes/copies.rst]: What I mean is "I'm not sure I understand it fully myself, so would love an explanation from someone else" 😊

Comment (jreback) [reply doc/source/getting_started/comparison/includes/copies.rst]: oh i c. check this blog by @TomAugspurger https://tomaugspurger.github.io/method-chaining.html

generaly `inplace` doesn't do a whole lot, is not idiomatic, and cause  copy/view issues.

**Pull request merged**